crates/ruff_linter/src/rules/pylint/rules/collapsible_else_if.rs

```diff
@@ -122,15 +126,30 @@ fn convert_to_elif(
         stylist,
     )?;
 
+    // If there's trivia, restore it
+    let trivia = if trivia_range.is_empty() {
+        None
+    } else {
+        let indented_trivia =
+            adjust_indentation(trivia_range, indentation, locator, indexer, stylist)?;
+        Some(Edit::insertion(
+            indented_trivia,
+            locator.line_start(else_clause.start()),
+        ))
+    };
+
     // Strip the indent from the first line of the `if` statement, and add `el` to the start.
     let Some(unindented) = indented.strip_prefix(indentation) else {
         return Err(anyhow::anyhow!("indented block to start with indentation"));
     };
     let indented = format!("{indentation}el{unindented}");
 
-    Ok(Fix::safe_edit(Edit::replacement(
-        indented,
-        locator.line_start(else_clause.start()),
-        inner_if_line_end,
-    )))
+    Ok(Fix::safe_edits(
+        Edit::replacement(
+            indented,
+            locator.line_start(else_clause.start()),
+            inner_if_line_end,
+        ),
+        trivia,
+    ))
```

Comment:
Interesting that this works because `safe_edits` accepts an `IntoIterator<item = Edit>` but this is an `Option`. I guess `None` is basically taken as an indicator that the iterator ends and it won't be collected into a vec.